README.md

```diff
@@ -208,6 +213,183 @@ try {
 
 <a name="help"/>
 
+<a name="in-process-linking"/>
+
+## Trace in-process asynchronous execution
+
+You can use the SDK to trace asynchronous in-process code execution. This might be useful if Dynatrace does not support the threading framework or specific asynchronous libraries. In-process-linking will only be used to link other services (Database-, Webrequests, ...) between thread or queuing boundaries, where currently no support is available.
```

Comment:
Dynatrace => OneAgent
where currently no support is available => something like currently not supported out of the box by OneAgent